dbt/include/firebolt/macros/adapters.sql

```diff
@@ -23,6 +23,9 @@
 
 
 {% macro firebolt__list_schemas(database) %}
```

Comment:
I like it! clever hack given that list_schemas is invoked with every run pretty much.... does it still work when threads=1?

Reply:
Yep, works great. Thank you @jerco!